### PR TITLE
Clean up franken-loop in data_loaders.py

### PR DESCRIPTION
**Pull request by nicolejkeeney**
Added descriptive code comments and removed legacy code from past average meteorological year related things.

### CODE REVIEW

**Review by vicford — APPROVED** [review]
Looks good to me!

File: climakitae/data_loaders.py [hunk 1]
```diff
@@ -410,6 +410,11 @@ def _get_data_one_var(selections, cat):
         # Get catalog subset for a set of user selections
         cat_subset = _get_cat_subset(selections=selections, cat=cat)
 
+        if len(cat_subset.df["institution_id"].unique()) == 1:
+            _institution = cat_subset.df["institution_id"].unique()[0]
+        else:
+            _institution = "Multiple"
```
Comment (nicolejkeeney) [inline climakitae/data_loaders.py:416]: Great addition, thanks!

**Review by nicolejkeeney — COMMENTED** [review]
Looks good to me! Nice changes.

**Review by vicford — APPROVED** [review]
Nice work, looks good to me

**Pull request merged**